napari/layers/surface/surface.py

```diff
@@ -27,22 +27,41 @@ class Surface(IntensityVisualizationMixin, Layer):
     """
     Surface layer renders meshes onto the canvas.
 
+    Note that surfaces may be colored by:
+        * texture (requires both texture and texcoords to be set)
+        AND
+            * vertex_colors (higher precedence)
+            OR
+            * vertex_values + colormap
```

Comment:
The indentation here is confusing... Maybe use brackets?

Reply:
And the texture is not required, right? If I read this I assume a texture is required...

Reply:
I agree this was confusing, thanks! I adapted the text from the PR description and I think it's more clear. Let me know what you think.